docs/api/adbc.md

```diff
@@ -123,3 +130,82 @@ StatementSetOption(&adbc_statement, ADBC_INGEST_OPTION_TARGET_TABLE, "AnswerToEv
 StatementBindStream(&adbc_statement, &arrow_stream, &adbc_error);
 StatementExecuteQuery(&adbc_statement, nullptr, nullptr, &adbc_error);
 ```
+
+### Python
+
+The first thing to do is to use `pip` and install the ADBC Driver manager.
+
+```shell
+pip install adbc_driver_manager
+```
+
+You will also need the `pyarrow` package to directly access Apache Arrow
+formatted result sets.
+```shell
+pip install pyarrow
+```
+
+The full documentation for the `adbc_driver_manager` package can be found
+[here](https://arrow.apache.org/adbc/current/python/api/adbc_driver_manager.html).
+
+As with C++, we need to provide initialization options 
+```python
+import adbc_driver_manager.dbapi
+conn = adbc_driver_manager.dbapi.connect(
+    driver="path/to/libduckdb.dylib",
+    entrypoint="duckdb_adbc_init",
+    db_kwargs={"path": "test.db"},
+)
+
+with conn.cursor() as cur:
+    pass
+
+conn.close()
+```
+
+It is also possible to leverage the duckdb Python library directly:
+```python
+import duckdb
+import adbc_driver_manager.dbapi
+conn = adbc_driver_manager.dbapi.connect(
+    driver=duckdb.__file__,
+    entrypoint="duckdb_adbc_init",
+    db_kwargs={"path": "test.db"}
+)
+
+with conn.cursor() as cur:
+    pass
+
+conn.close()
+```
+
+Now that we have a connection initialized, we can initialize a cursor
```

Comment:
1) Please weave these two examples together. Currently, the first example is just opens a connection and closes it immediately. The second one has `...` for where the imports should happen `conn` should be initialized.
2) Please print something (`print(tbl)`, `print(df)`, ...) to stdout so the user can see that something is happening.
3) The `fetchone`/`fetchall` functions should be mentioned in the text, not in comments.
4) Maybe push the `conn` initialization into the same `with` block like so?

```py
with adbc_driver_manager.dbapi.connect(
        driver=duckdb.__file__,
        entrypoint="duckdb_adbc_init",
        db_kwargs={"path": "test.db"}
    ) as conn, conn.cursor() as cur:
    cur.execute("SELECT 42")
    # fetch as a pyarrow table
    tbl = cur.fetch_arrow_table()
    print(tbl)
```

Reply:
Done